app/konnect/dev-portal/applications/dynamic-client-registration/okta.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Configuring Okta for Dynamic Client Registration
+no_version: true
+toc: true
+breadcrumb: Okta
+content-type: how-to
+---
+
+
+## Prerequisites
+
+To use Okta as an external client provider for Dynamic Client Registration (DCR), create or use an existing Okta account [here](http://developer.okta.com). Once you have an account, you will need to create a token, find the Issuer URL, and optionally set any scopes or claims.
```

Comment:
Should we mention that it is only for `tier === enterprise` only?

Reply:
Done